dockerfiles/init/modules/compose/templates/docker-compose.yml.erb

```diff
@@ -101,18 +101,25 @@ services:
   ########################
   postgres:
     image: <%= ENV["IMAGE_POSTGRES"] %>
+    command: [ "/var/lib/pgsql/init-che-user-and-run.sh" ]
     environment:
-      - POSTGRES_USER=keycloak
-      - POSTGRES_PASSWORD=keycloak
-      - POSTGRES_DB=keycloak
+      - POSTGRESQL_USER=keycloak
+      - POSTGRESQL_PASSWORD=keycloak
+      - POSTGRESQL_DATABASE=keycloak
+      - CHE_POSTGRES_USERNAME=<%= scope.lookupvar('postgres::che_pg_username') %>
+      - CHE_POSTGRES_PASSWORD=<%= scope.lookupvar('postgres::che_pg_password') %>
+      - CHE_POSTGRES_DATABASE=<%= scope.lookupvar('postgres::che_pg_database') %>
+ 
+  
     volumes:
-      - '<%= scope.lookupvar('che::che_instance') -%>/data/postgres:/var/lib/postgresql/data'
-      - '<%= scope.lookupvar('che::che_instance') -%>/config/postgres/init-che-user.sh:/docker-entrypoint-initdb.d/init-che-user.sh'
+      - '<%= scope.lookupvar('che::che_instance') -%>/data/postgres:/var/lib/pgsql/data'
+      - '<%= scope.lookupvar('che::che_instance') -%>/config/postgres/init-che-user.sh:/var/lib/pgsql/init-che-user.sh'
+      - '<%= scope.lookupvar('che::che_instance') -%>/config/postgres/init-che-user-and-run.sh:/var/lib/pgsql/init-che-user-and-run.sh'
     ports:
       - '5432:5432'
     restart: always
     healthcheck:
-        test: "pg_isready -h localhost -p 5432 -U postgres"
```

Comment:
why pg_isready does not work here?

Reply:
This PR now uses the official, CentOS-based, Postgres docker image of the Openshift Online catalog (https://console.starter-us-east-2.openshift.com/console/project/dfestal/create/fromtemplate?template=postgresql-persistent&namespace=openshift), whose sources are here: https://github.com/sclorg/postgresql-container/tree/master/9.5.
This image is compatible with *bothù simple docker and Openshift deployments, which is certainly a good thing for maintenance.

Since it's based on CentOS and uses SCL, `pg_isready` is not directly available for the health check. Additionaly, I thought it was a good thing to use the same health-check as the one officially used for this image on Openshift Online, as defined here: https://github.com/openshift/library/blob/master/official/postgresql/templates/postgresql-persistent.json#L159